static/src/javascripts/projects/membership/digitalpack-tab.js

```diff
@@ -0,0 +1,102 @@
+define([
+    'bean',
+    'common/utils/$',
+    'common/utils/ajax',
+    'common/utils/config',
+    'membership/formatters',
+    'membership/payment-form'
+], function (bean, $, ajax, config, formatters, PaymentForm) {
+
+    var PACKAGE_COST = '.js-dig-package-cost',
+        CARD_CHANGE_SUCCESS_MSG = '.js-dig-card-change-success-msg',
+        PAYMENT_FORM = '.js-dig-card-details',
+        SUBSCRIBER_ID = '.js-dig-subscriber-id',
+        REMAINING_TRIAL_LENGTH = '.js-dig-remaining-trial-length',
+        REMAINING_TRIAL_LENGTH_CONTAINER = '.js-dig-remaining-trial-length-container',
+        PACKAGE_CURRENT_RENEWAL_DATE = '.js-dig-current-renewal-date',
+        PACKAGE_CURRENT_PERIOD_END = '.js-dig-current-period-end',
+        PACKAGE_CURRENT_PERIOD_START = '.js-dig-current-period-start',
+        PACKAGE_NEXT_PAYMENT_DATE = '.js-dig-next-payment-date',
+        PACKAGE_NEXT_PAYMENT_PRICE = '.js-dig-next-payment-price',
+        PACKAGE_NEXT_PAYMENT_CONTAINER = '.js-dig-next-payment-container',
+        PACKAGE_INTERVAL = '.js-dig-plan-interval',
+        DETAILS_JOIN_DATE = '.js-dig-join-date',
+        NOTIFICATION_CANCEL = '.js-dig-cancel-tier',
+        DIGITALPACK_DETAILS = '.js-dig-details',
+        DIGITALPACK_PRODUCT = '.js-dig-product',
+        UP_SELL = '.js-dig-up-sell',
+        DIG_INFO = '.js-dig-info',
+        LOADER = '.js-dig-loader',
+        IS_HIDDEN_CLASSNAME = 'is-hidden',
+        stripeForm = new PaymentForm(PAYMENT_FORM, CARD_CHANGE_SUCCESS_MSG, '/me/digitalpack-update-card');
+
+    function fetchUserDetails() {
+        ajax({
+
+            url: config.page.userAttributesApiUrl + '/me/mma-digitalpack',
+            crossOrigin: true,
+            withCredentials: true,
+            method: 'get'
+        }).then(function (resp) {
```

Comment:
Well this behaviour is consistent with the membership version of this page
I do agree we should handle errors though